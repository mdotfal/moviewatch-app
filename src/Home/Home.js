import React, { Component } from 'react';
import Header from '../Header/Header';
import ItemForm from '../ItemForm/ItemForm';

class Home extends Component {
  render() {
    return (
      <div>
        <Header/>
        <section>
          <div>
            Recommend a title
          </div>
          <ItemForm />
        </section>
      </div>
    );
  }
};

export default Home;