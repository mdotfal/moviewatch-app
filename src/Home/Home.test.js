import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from './Home';

describe( `Home component`, () => {
  it(Â `renders without errors`, () => {
    const wrapper = shallow( <Home /> )
    expect( toJson( wrapper )).toMatchSnapshot()
  })
})