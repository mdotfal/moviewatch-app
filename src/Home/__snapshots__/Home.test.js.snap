// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders without errors 1`] = `
<div>
  <Header />
  <section
    className="home"
  >
    <div>
      Recommend a title
    </div>
  </section>
</div>
`;
